interfaces/Portal/src/app/app.module.ts

```diff
@@ -116,7 +116,7 @@ if (environment.use_sso_azure_entra) {
       new PublicClientApplication({
         auth: {
           clientId: environment.azure_ad_client_id,
-          authority: `https://${environment.azure_ad_tenant_id}.ciamlogin.com/${environment.azure_ad_tenant_id}/v2.0`,
+          authority: 'https://login.microsoftonline.com/common',
```

Comment:
```suggestion
          authority: `${environment.azure_ad_url}/common`,
```

This would be the same, right? 
Are there actually other hostnames/URLs that could be used for this Azure Entra service?

Reply:
Done.